sdk/databoundaries/arm-databoundaries/sample.env

```diff
@@ -0,0 +1,4 @@
+# App registration secret for AAD authentication
+AZURE_CLIENT_SECRET=
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID= 
```

Comment:
can you remove this? I wonder why we didn't change it in the sample env codegen pr?

Reply:
we need to release a new version for codegen to cover the sample env change in hlc